curations/npm/npmjs/@microsoft/applicationinsights-channel-js.yaml

```diff
@@ -7,3 +7,6 @@ revisions:
   3.3.4:
     licensed:
       declared: MIT
+  3.3.5:
```

Comment:
Is it possible to set wider version ranges?

Reply:
Are you referring to curating a range instead of a version number? If so, that is a requested feature but I don't think there is any planned work on it.